app/js/ui.js

```diff
@@ -28,11 +27,13 @@ $(document).ready(function() {
     $(".dropdown-button").dropdown();
 });
 
+//Converts svg to png and downloads it as 'cartogram.png'
 function download_png() {
     var svg = d3.select('svg');
```

Comment:
Get rid of the comment, and do camel case (downloadPng()) instead. No need for a comment if the code explains it. Maybe call it downloadCartogramPNG instead?